src/deploy/functions/params.ts

```diff
@@ -605,7 +605,7 @@ async function promptText<T extends RawParamValue>(
       return promptText<T>(prompt, input, resolvedDefault, converter);
     }
   }
-  const converted = converter(res);
+  const converted = converter(typeof res === "string" ? res : res.toString());
```

Comment:
fun fact: `string` also implements `toString` so that you can avoid ternaries like this.

Reply:
You know, I knew that, but I apparently didn't _know_ that.